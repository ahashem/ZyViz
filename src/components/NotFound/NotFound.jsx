import React from 'react';

const NotFound = () => (
  <div>
    <h1 className="header">
      404
    </h1>
    <h6 className="sub-header">
      This page does not exist
    </h6>
    <p className="uk-margin-left uk-text-middle uk-text-muted">
      go back to Home ->
    </p>
  </div>
);

export default NotFound;
