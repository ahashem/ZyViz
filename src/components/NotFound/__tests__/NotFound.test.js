import { shallow } from 'enzyme/build';
import React from 'react';
import NotFound from '../NotFound';


describe('<NotFound /> Component', () => {
  it('should render children and match snapshot', () => {
    const rendered = shallow(<NotFound/>);

    expect(rendered).toMatchSnapshot();
  });
});
