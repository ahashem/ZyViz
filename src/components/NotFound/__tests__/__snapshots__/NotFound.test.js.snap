// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Component should render children and match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="header"
        >
          404
        </h1>,
        <h6
          className="sub-header"
        >
          This page does not exist
        </h6>,
        <p
          className="uk-margin-left uk-text-middle uk-text-muted"
        >
          go back to Home -&gt;
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
          "className": "header",
        },
        "ref": null,
        "rendered": "404",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "This page does not exist",
          "className": "sub-header",
        },
        "ref": null,
        "rendered": "This page does not exist",
        "type": "h6",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "go back to Home ->",
          "className": "uk-margin-left uk-text-middle uk-text-muted",
        },
        "ref": null,
        "rendered": "go back to Home ->",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="header"
          >
            404
          </h1>,
          <h6
            className="sub-header"
          >
            This page does not exist
          </h6>,
          <p
            className="uk-margin-left uk-text-middle uk-text-muted"
          >
            go back to Home -&gt;
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
            "className": "header",
          },
          "ref": null,
          "rendered": "404",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "This page does not exist",
            "className": "sub-header",
          },
          "ref": null,
          "rendered": "This page does not exist",
          "type": "h6",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "go back to Home ->",
            "className": "uk-margin-left uk-text-middle uk-text-muted",
          },
          "ref": null,
          "rendered": "go back to Home ->",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
